uber/models.py

```diff
@@ -1654,6 +1654,14 @@ class MPointsForCash(MagModel):
     when        = Column(UTCDateTime, default=lambda: datetime.now(UTC))
 
 
+class PromoCode(MagModel):
+    price           = Column(Integer, default=0)
+    uses            = Column(Integer, default=1)
+    code            = Column(UnicodeText)
+    users           = Column(UnicodeText, default="")
```

Comment:
`used = session.query(PromoCodeUsages).filter(PromoCodeUsages.attendee_id == params['edit_id']).first()
                            if used is None:`

how I'm currently handling it.